docs/src/content/docs/zh-cn/guides/components.mdx

```diff
@@ -58,23 +58,24 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 你可以使用 `<Tabs>` 和 `<TabItem>` 组件显示一个选项卡界面。
 每个 `<TabItem>` 必须有一个 `label` 来显示给用户。
+使用可选的`<icon>`属性在标签旁包含一个[Starlight内置图标](#所有图标)。
```

Comment:
```suggestion
使用可选的`icon`属性在标签旁包含一个[Starlight内置图标](#所有图标)。
```

One last suggestion regarding your last changes and I think we should be good to merge 🎉

Reply:
Thanks for your patience.